TF_Tooling_Web/src/tests/testUtils.ts

```diff
@@ -15,3 +16,17 @@ export const mockTfInputAction = (input?: Partial<TouchFreeInputAction>) =>
             input?.ProgressToClick ?? 0
         )
     );
+
+export const mockTfPluginInputAction = (input?: Partial<TouchFreeInputAction>) =>
+    InputActionManager.HandleInputAction(
+        new TouchFreeInputAction(
```

Comment:
:eyes: Looks like we might end up using this default structure frequently. Could it be functionalised to use across the two mocks?